<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyApp</class>
 <widget class="QMainWindow" name="MyApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Slider</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_image">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>401</width>
      <height>341</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap resource="../Archivos/Recursos.qrc">:/ejercicios/image_4.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="selectorImagen">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
            border: 1px solid #999999;
            height: 8px;
            background: rgb(170, 170, 255);
            margin: 2px 0;
        }
        QSlider::handle:horizontal {
            background: white;
            border: 1px solid #5c5c5c;
            width: 18px;
            margin: -2px 0;
            border-radius: 3px;
        }</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);
</string>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 127)
</string>
    </property>
    <property name="text">
     <string>Edad: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_edad">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 18px;
background-color: white;
border:2px solid rgb(170, 170, 255);
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_nombre">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 18px;
background-color: white;
border:2px solid rgb(170, 170, 255);
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 127)
</string>
    </property>
    <property name="text">
     <string>Juguete:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_juguete">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 18px;
background-color: white;
border:2px solid rgb(170, 170, 255);
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_iniciar">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: linear-gradient(135deg, rgb(0, 0, 0), rgb(25, 25, 25));
    color: rgb(0, 255, 255);
    border-radius: 25px;
    border: 2px solid rgb(0, 255, 255);
    padding: 10px;
    font: bold 10pt &quot;OCR A Extended&quot;;
    transition: background 0.3s, transform 0.3s, box-shadow 0.3s;
    box-shadow: 0 0 10px rgba(0, 255, 255, 0.7);
}

 QPushButton:hover {
                background-color: cyan;
                color: white;
            }
            QPushButton:pressed {
                background-color: cyan;
                color: white;
            }</string>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_detener">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>500</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background: linear-gradient(135deg, rgb(0, 0, 0), rgb(25, 25, 25));
    color: rgb(0, 255, 255);
    border-radius: 25px;
    border: 2px solid rgb(0, 255, 255);
    padding: 10px;
    font: bold 10pt &quot;OCR A Extended&quot;;
    transition: background 0.3s, transform 0.3s, box-shadow 0.3s;
    box-shadow: 0 0 10px rgba(0, 255, 255, 0.7);
}

 QPushButton:hover {
                background-color: cyan;
                color: white;
            }
            QPushButton:pressed {
                background-color: cyan;
                color: white;
            }</string>
    </property>
    <property name="text">
     <string>Detener</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_intervalo">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 18px;
background-color: white;
border:2px solid rgb(170, 170, 255);
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>390</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 127)
</string>
    </property>
    <property name="text">
     <string>Introduce el intervalo (seg):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_image_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap resource="../Archivos/Recursos.qrc">:/ejercicios/LOGOUAT.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_image_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap resource="../Archivos/Recursos.qrc">:/ejercicios/LOGOFACULTAD.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_image_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="pixmap">
     <pixmap resource="../Archivos/Recursos.qrc">:/ejercicios/imgEq5.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Archivos/Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
